shell/edgexfoundry-device-virtual-go-docker-push.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+# Ensure we fail the job if any steps fail
+set -e -o pipefail
+
+# Required parameters:
+#
+#  VERSION: Set this value based on the VERSION file in the project root.
+#  DEPLOY_TYPE: Can be `snapshot`, `staging` or`release`
```

Comment:
Where is DEPLOY_TYPE defined?

Reply:
DEPLOY_TYPE is defined in [edgex-templates-go.yaml](https://github.com/edgexfoundry/ci-management/blob/master/jjb/edgex-templates-go.yaml#L168)